src/modules/todo/todo.controller.ts

```diff
@@ -11,6 +11,8 @@ import { TodoService } from './todo.service.ts';
 import { Request } from 'express';
 import { APIContainer } from '../../inversify.config.ts';
 import * as express from 'express';
+import { AppError } from '../errors/AppError.ts';
+import httpStatus from 'http-status';
```

Comment:
It looks like you imported this packages but not using it.

Reply:
Add  eslint-remove-unused-imports package
https://github.com/ArtyomZayarny/todo-api/pull/14